spring-cloud-gcp-dependencies/pom.xml

```diff
@@ -274,13 +276,32 @@
 				<version>${cloud-sql-socket-factory.version}</version>
 			</dependency>
 
+			<!-- Spring Boot 2.7 no longer manages this dependency.
+			     Replace with MariaDB driver once it's supported.
+			     https://github.com/GoogleCloudPlatform/cloud-sql-jdbc-socket-factory/issues/990
+			 -->
+			<dependency>
+				<groupId>dev.miku</groupId>
+				<artifactId>r2dbc-mysql</artifactId>
+				<version>${r2dbc-mysql-driver.version}</version>
+			</dependency>
+
 			<!-- spring-cloud-gcp-starter-sql-postgres-r2dbc -->
 			<dependency>
 				<groupId>com.google.cloud.sql</groupId>
 				<artifactId>cloud-sql-connector-r2dbc-postgres</artifactId>
 				<version>${cloud-sql-socket-factory.version}</version>
 			</dependency>
 
+			<!-- Spring Boot 2.7 no longer manages this dependency.
+     			 Replace with org.postgres group ID once Spring Boot 2.6 is out of support.
+ 			-->
+			<dependency>
+				<groupId>io.r2dbc</groupId>
+				<artifactId>r2dbc-postgresql</artifactId>
+				<version>${r2dbc-postgres-driver.version}</version>
+			</dependency>
```

Comment:
Will our tests still continue to verify Spring Boot 2.6?

Reply:
I thought about introducing a new GHA job, like we had with 2.5 support, but 2.6 is [going out of support in a month](https://spring.io/projects/spring-boot#support)...

Reply:
Still might be worth keeping around for a few months. I'll look into it in a separate PR.

Reply:
#1292 to track.

Reply:
I see! It is worth it for the sake of making sure there aren't any breaking changes, if doesn't take too much time. Most importantly, I think it is also worth testing Spring Boot 2.7 with the new postgresql driver (with the `org.postgres` coordinates) since we document this.

Reply:
As discussed; it's either 2.6 testing or the exclude/include-new-driver flow. Chose 2.6 integration testing.